metabase/metabase
# extending table component features

### Description

Add a few new features to out existing simple table component:
- `ifEmpty` prop for when there are no rows
- `formatValueForSorting` to handle sorting complex values
- pagination controls 
- `cols` prop to add column groups
- decouple the table from the default sorting (useful for API driven tables)

### How to verify
Currently, this table is only used for API Keys and Upload Management, Ensure that things work as expected in those areas

### Checklist
